### DISCUSSION TITLE
KiCAD 8.0 ERC reports 102 warnings and 6 errors

### DISCUSSION THREAD

**Discussion by vitasam** [markadev/AppleII-VGA#46] [General]
Hi, KiCAD 8.0 ERC reports 102 warnings and 6 errors om VGA card schematics

**Comment by vitasam** [comment]
Most of warnings and errors are solvable, but one warning is about missed "MCU_RaspberryPi_and_Boards:RPi_Pico_SMD_TH" library - where to get it?

**Reply by markadev** [reply]
Hey there, yeah unfortunately a lot of the warnings are bogus due to the ERC not understanding that some of the tristate pins are hard-wired to be inputs.  The Pico symbol and footprints are from this library: https://github.com/ncarandini/KiCad-RP-Pico

**Reply by vitasam** [reply]
Yes. One of ERC errors is due to the fact that the pin 38 of RPi Pico (GND) is defined as "Bidirectional":

![image](https://github.com/user-attachments/assets/589a2b66-fbba-4383-a4aa-2511fcb7464e)
